src/null-safety/migration-guide.md

```diff
@@ -430,34 +430,23 @@ If so, revert your code changes before using the migration tool again.
 We encourage you to publish packages as prereleases
 as soon as you migrate:
 
-* [Set the SDK constraints to the tested beta version.](#sdk-constraints)
 * [Set the package version to indicate a breaking change.](#package-version)
+* [Publish the package as a prerelease if development is still in progress.][]
 
-### SDK constraints
```

Comment:
This section was confusing and I'm not sure it's necessary now that 2.12 is out of beta.

Reply:
Perhaps we should still say to set the SDK constraint to the lowest tested SDK version, though?